add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
Observations: The application lacks unit tests for the main functionalities.
Recommendations: Implement unit tests using a framework like PHPUnit to ensure the robustness of the application. For example, test the database connection, CRUD operations, and form submissions.

Example test case:

```php
use PHPUnit\Framework\TestCase;

class BookTest extends TestCase {
    public function testAddBook() {
        $db = new Database();
        $result = $db->query("INSERT INTO books (title, author, year) VALUES ('Test Title', 'Test Author', 2023)");
        $this->assertTrue($result);
    }
}
```